src/main/java/io/gravitee/common/environment/GraviteeEnvironment.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.common.environment;
+
+public interface GraviteeEnvironment {
+    boolean containsProperty(String var1);
+
+    String getProperty(String var1);
+
+    String getProperty(String var1, String var2);
+
+    <T> T getProperty(String var1, Class<T> var2);
+
+    <T> T getProperty(String var1, Class<T> var2, T var3);
+
+    String getRequiredProperty(String var1) throws IllegalStateException;
```

Comment:
I think this method and the next ones are not useful and looks it's been copied directly from the Environement class.